docs/workloads/azure/data/intro_data_azure.md

```diff
@@ -20,7 +20,7 @@ a framework for accelerating the deployment of a production-ready modern data pl
 
 ![Ensono Stacks Data Overview](images/stacks-data-overview.png)
 
-1. Use the [Ensono Stacks CLI](../../../stackscli/about) to generate a new data platform project.
+1. Use the [Ensono Stacks CLI](https://stacks.ensono.com/docs/stackscli/about) to generate a new data platform project.
```

Comment:
Can you revert this line back - we can use the relative path in the full Stacks docs.